package.json

```diff
@@ -40,7 +40,7 @@
     "@nx/rollup": "16.5.5",
     "@nx/web": "16.5.5",
     "@nx/workspace": "16.5.5",
-    "@openfeature/js-sdk": "^1.3.1",
+    "@openfeature/js-sdk": "^1.4.1",
```

Comment:
Did you intend to bump this version? If so, make sure to bump the minimum peer version [here](https://github.com/open-feature/js-sdk-contrib/blob/main/libs/providers/flagd/package.json#L10).

Reply:
This is just for local dev, as discussed.